oxidation/libparsec/crates/crypto/src/rustcrypto/sequester.rs

```diff
@@ -44,15 +37,25 @@ impl TryFrom<&[u8]> for SequesterPrivateKeyDer {
 }
 
 impl EnforceDeserialize for SequesterPrivateKeyDer {
-    const SIZE_IN_BITS: usize = Self::SIZE_IN_BITS;
     const ALGORITHM: &'static [u8] = b"RSAES-OAEP-XSALSA20-POLY1305";
+
+    fn size_in_bytes(&self) -> usize {
+        self.size_in_bytes()
+    }
 }
 
 impl SequesterPrivateKeyDer {
-    #[cfg(test)]
-    pub const SIZE_IN_BITS: usize = 1024;
-    #[cfg(not(test))]
-    pub const SIZE_IN_BITS: usize = 4096;
+    pub fn generate_pair(size_in_bits: usize) -> (Self, SequesterPublicKeyDer) {
+        let priv_key = RsaPrivateKey::new(&mut rand_08::thread_rng(), size_in_bits)
+            .expect("Invalid key size, should be used only for test");
```

Comment:
```suggestion
            .expect("Failed to generate a new RSA private key");
```
The key size is not the only error ;)

Reply:
How ? It's the only one input 🤔